projects/Mallard/src/components/article/html/render.ts

```diff
@@ -156,6 +155,8 @@ export const render = (
         headerProps?: ArticleHeaderProps & { type: ArticleType }
     },
 ) => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const { imageSize } = useImageSize()
```

Comment:
Should we not make this a custom hook like `useRenderedHTML` in order to avoid the warning?

Reply:
We could also put the memoization logic (that is wrapping this call) in there if we do that.

Reply:
This error came because of the name of the function. As far as im aware it should be fine there.

Reply:
I think we should probably change the name just because, as a hook, it can't then be called conditionally. This will get picked up by our linter if it starts with `use` if someone does that in future. As it stands we'll just get a runtime error if it gets called conditionally.